src/cmd-buildupload

```diff
@@ -124,7 +127,8 @@ def s3_upload_build(args, builddir, bucket, prefix):
             uploaded.add(bn)
             continue
 
-        s3_target_exists = s3_check_exists(bucket, s3_path, args.dry_run)
+        s3_target_exists = s3_check_exists(bucket, s3_path, args.dry_run,
+                            endpoint_url=args.endpoint_url)
```

Comment:
Minor: indentation looks off here.

Reply:
Would you like continuations to line-up open parenthesis?

Reply:
Yeah, I think that's the convention we use.